### PR TITLE
feat: add IPath.GetFullPath(string,string)

### PR DESCRIPTION
**Pull request by Ibrahim-Islam**
#654

### CODE REVIEW

File: src/System.IO.Abstractions/IPath.cs [hunk 1]
```diff
@@ -36,6 +36,10 @@ public interface IPath
         string GetFileNameWithoutExtension(string path);
         /// <inheritdoc cref="Path.GetFullPath(string)"/>
         string GetFullPath(string path);
+#if NETSTANDARD2_1
```
Comment (fgreinacher) [inline src/System.IO.Abstractions/IPath.cs:39]:
```suggestion
#if FEATURE_ADVANCED_PATH_OPERATIONS
```

File: src/System.IO.Abstractions/PathBase.cs [hunk 2]
```diff
@@ -63,6 +63,11 @@ protected PathBase(IFileSystem fileSystem)
         /// <inheritdoc cref="Path.GetFullPath(string)"/>
         public abstract string GetFullPath(string path);
 
+#if NETSTANDARD2_1
```
Comment (fgreinacher) [inline src/System.IO.Abstractions/PathBase.cs:66]:
```suggestion
#if FEATURE_ADVANCED_PATH_OPERATIONS
```

File: src/System.IO.Abstractions/PathWrapper.cs [hunk 3]
```diff
@@ -83,6 +83,13 @@ public override string GetFullPath(string path)
             return Path.GetFullPath(path);
         }
 
+# if NETSTANDARD2_1
```
Comment (fgreinacher) [inline src/System.IO.Abstractions/PathWrapper.cs:86]:
```suggestion
#if FEATURE_ADVANCED_PATH_OPERATIONS
```

**Review by fgreinacher — CHANGES_REQUESTED** [review]
Thanks a lot! Let's use the feature-based constants for consistency

**Comment by Ibrahim-Islam** [comment]
Would also really appreciate if you can label this pr `hacktoberfest-accepted`

**Comment by fgreinacher** [comment]
Sure, thanks a lot!

**Pull request merged**